cypress/integration/pages/home.spec.js

```diff
@@ -46,12 +46,20 @@ context('Index page', () => {
       cy.get('[data-testid=food-needs-v-halal-checkbox]').click();
       cy.get('[data-testid=food-needs-v-vegetarian-checkbox]').click();
 
+      // it calculates the distance, there may be a considerable network delay
+      cy.wait(3000)
```

Comment:
I prefer "wait until thing happens" rather than "wait X amount of time", since it's less flaky.

Cypress doesn't support "wait until this resource on the page is loaded" like Capybara does, but you might be able to [wait for the request to come back](https://docs.cypress.io/guides/guides/network-requests.html#Waiting) - it's not something I've done in Cypress before, though.

Up to you if you want to spend time on this - I'm happy to add it now and change later if it turns out to be flaky.

Reply:
I have added it now

Reply:
I am struggling with this one, I will have to leave it behind for now